src/actions/index.js

```diff
@@ -43,28 +38,26 @@ export const login = values => (dispatch) => {
     });
 };
 
-export const rsvpResponse = (user, form, push) => (dispatch) => {
-  // rsvpRef
-  //   .doc(user.uid)
-  //   .set(form)
-  //   .then(() => {
-  //     dispatch({ type: types.UPDATE_RSVP, rsvpVal: true });
-  //     push('/dashboard');
-  //   });
+export const rsvpResponse = () => (dispatch) => {
+  dispatch({ type: types.UPDATE_RSVP, rsvpVal: true });
 };
 
-export const forgotPassword = (values, callback) => (dispatch) => {
+export const forgotPassword = values => (dispatch) => {
```

Comment:
Put back the callback call

It will be needed so we can display to the user:
"If the account exists, you will receive an email to reset your password."

Without the callback, the <Dialog> component won't display. More info here: https://material-ui.com/api/dialog/